src/components/patterns/ski-scotland/ski-scotland-card/SkiScotlandCard.vue

```diff
@@ -0,0 +1,668 @@
+                                size="xs"
+                                class="mr-2"
+                            />
+                            <span data-test="vs-ski__closed-label">
+                                {{ summaryClosedLabel }}
+                            </span>
+                        </VsTableDataCell>
+                        <VsTableDataCell
+                            v-if="runs.length"
+                        >
+                            {{ statusSummary.runs.closed }}/{{ runs.length }}
+                        </VsTableDataCell>
+                        <VsTableDataCell>
+                            {{ statusSummary.lifts.closed }}/{{ lifts.length }}
+                        </VsTableDataCell>
+                    </VsTableRow>
+                    <VsTableRow
+                        v-if="statusSummary.runs.onHold || statusSummary.lifts.onHold"
+                    >
+                        <VsTableDataCell>
+                            <VsIcon
+                                name="hourglass"
+                                size="xs"
+                                class="mr-2"
+                            />
+                            <span data-test="vs-ski__on-hold-label">
+                                {{ summaryOnHoldLabel }}
+                            </span>
+                        </VsTableDataCell>
+                        <VsTableDataCell
+                            v-if="runs.length"
+                        >
+                            {{ statusSummary.runs.onHold }}/{{ runs.length }}
+                        </VsTableDataCell>
+                        <VsTableDataCell>
+                            {{ statusSummary.lifts.onHold }}/{{ lifts.length }}
+                        </VsTableDataCell>
+                    </VsTableRow>
+                </VsTableBody>
+                <VsTableFooter>
+                    <VsTableRow>
+                        <VsTableDataCell
+                            colspan="3"
+                            role="cell"
+                        >
+                            <span data-test="vs-ski__last-updated-label">
+                                {{ lastUpdatedLabel }}: {{ lastUpdate }}
+                            </span>
+                        </VsTableDataCell>
+                    </VsTableRow>
+                </VsTableFooter>
+            </VsTable>
+            <div class="vs-ski-scotland-card__button-holder">
+                <VsButton
+                    v-if="pisteMapLink"
+                    variant="secondary"
+                    :href="pisteMapLink"
+                    size="sm"
+                >
+                    {{ pisteMapLabel }}
+                </VsButton>
+                <VsButton
+                    v-else
+                    disabled
+                    size="sm"
+                >
+                    {{ pisteMapLabel }}
+                </VsButton>
+            </div><!--
```

Comment:
Is there a comment missing here?

Reply:
No, it's an awkward one where the whitespace in the file causes some issues for the inline-block layout by putting an nbsp in the output. It could be fixed by setting the container font size to 0 or just putting the opening div straight after the closing one but this keeps the indentation consistent with everything else (and I think it all just then gets stripped out in the build/minification anyway to be </div><div> rather than </div> <div>)